Експериментални CSS промени

AndrewRadev opened the pull request:
Предлагам някои промени в заглавията. Добавил съм и ключалка на голямото заглавие на текста. Не настоявам всички да се включат, просто предлагам:

## Index:
![Index](https://user-images.githubusercontent.com/124255/35775595-62ba7ab8-0994-11e8-846b-35ee4f79be32.png)

## Статия:
![Статия](https://user-images.githubusercontent.com/124255/35775596-62e8be14-0994-11e8-9cb1-6b532201b677.png)

AndrewRadev committed:
Various experimental style changes

kunev commented:
Ключалката ми е малко two parts, щото ми навява асоциации с running meme-а, "it has a padlock on it, so it must be safe!" 😄. Факт е обаче, че отделя доста добре „главното заглавие“ от останалите елементи по страницата, така че съм ок.

Подчертания heading е супер, много ми харесва. Тогава е добре или да се съгласим да ползваме `h2` навсякъде, т.е. без nesting на heading-и, или правилото да е поне за `h2` и `h3` (`h4` може би няма логика да се ползва за каквото и да било).

AndrewRadev commented:
Да, и аз се чудя за ключалката, така че не съм сигурен какво да я правим. Притеснява ме и дали ще може да се рендерира на по-стари браузъри. Не знам.

Иначе, генерално се чудя дали няма много логика да вкъкваме тук-таме emoji-та и/или fontawesome иконки, примерно в някакви bullet-и, като visual aid (📱 , 💻 , 💼  (служебни данни) , 🏠 (лични данни)). Предвид, че нямаме дизайнери, може би ще е минималното, което може да се постигне. Ще пробвам за някоя статия да направя такава версия, паралелно, да видим как ще изглежда.

> Тогава е добре или да се съгласим да ползваме h2 навсякъде, т.е. без nesting на heading-и, или правилото да е поне за h2 и h3 (h4 може би няма логика да се ползва за каквото и да било).

Да, не бих nest-вал заглавия. Bullet-ите помагат за групирани неща, а ако имаме под-секции, може би просто си заслужава да се реструктурира да бъдат главни секции. Правилото in general за h1 е да има само един на страницата, btw, затова го махнах от "логото".

AndrewRadev committed:
Merge branch 'master' into style-changes

* master: (21 commits)
  Remove overemphasising in intro
  Use h2 for section titles
  Fix spelling and typos
  Fix messed up paragraph wording and left over line
  Reorganize the last section to improve readability
  Style fixes on the acceptable risk section
  Add new section on defence cost
  Make the consequences section more concise
  Scrape most of the threat probability section
  Fix paragraph about social media account acccess
  Reword and reformat personal vs work section
  Shorten the section on possible threats
  Further shorten the intro
  Remove useless sentance in the end of the intro
  Refactor so the inital two examples stand out more
  Shorten threat modeling example questions
  Reorder intro to start off with The Question
  Redo section on personal vs. work information
  Redo section on possible threats
  Redo intro so it sounds less formal
  ...

kunev commented:
За останалите емоджита съм доста сигурен, че е по-добре без. Ако някой ден се случи така, че човек с правилния вид въображение и способности да визуализира части от текстовете се включи, това ще е решението на проблема, за който емоджитата биха били по-скоро workaround.

kunev commented:
Не се бях замислял много за влагането на заглавия, определено ми харесва твърдото правило всичко да е само `h2` и при всяка „нужда“ от влагане да се постига нужния ефект с bullet-и.

AndrewRadev committed:
Font change

This is a rather large change, but it only uses native fonts, and the
one used in the theme seems to be wonky for cyrillic on Windows.

AndrewRadev committed:
Margin tweaks

kunev commented:
Супер! Оправил си и нерендериращата се ключалка, така че сме go!
